packages/app/components/productCategories/CategoryItem.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { ImageSourcePropType } from 'react-native';
+import { TouchableOpacity } from 'react-native-gesture-handler';
```

Comment:
```suggestion
import { ImageSourcePropType, TouchableOpacity } from 'react-native';
```
This doesn't work on android. See this: https://stackoverflow.com/questions/53548361/react-native-touchableopacity-onpress-not-working-on-android